packages/angular/cli/lib/cli/index.ts

```diff
@@ -15,6 +15,7 @@ import { ngDebug } from '../../src/utilities/environment-options';
 import { writeErrorToLogFile } from '../../src/utilities/log-file';
 
 export { VERSION } from '../../src/utilities/version';
+export { EventCustomMetric, EventCustomDimension } from '../../src/analytics/analytics-parameters';
```

Comment:
Are these new exports used anywhere or are they for third-party usage?

Reply:
Actually it was for the validation script, but I can deep import.